javascript-modal/styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+button {
+  font-size: 1.9rem;
+  border-radius: 6px;
+  outline: none;
+  color: rgb(245, 245, 245);
+  border: none;
+  cursor: pointer;
+}
+
+.main-container > button {
+  margin: 1rem;
+  padding: 0.7rem 2.9rem;
+  background-color: #665de6;
+}
+
+.overlay {
+  background: rgba(0, 0, 0, 0.75);
+}
+
+.modal-container {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1;
+}
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 550px;
+  height: 285px;
+  background-color: rgb(247, 247, 247);
+  border-radius: 5px;
+}
+
+.modal > h1,
+.modal > button {
+  margin: 1.8rem 0;
+}
+
+.modal > h1 {
+  font-size: 2rem;
+}
+
+.modal > button {
+  padding: 0.5rem 2.1rem;
+  background-color: rgb(255, 81, 0);
+  text-transform: uppercase;
+}
+
+.show {
+  visibility: visible;
+}
+
+.hidden {
+  visibility: hidden;
+}
```

Comment:
I would make this `display: none` instead. `visibility` still leaves its elements with size on the page, which is typically not desirable.